service/src/main/kotlin/app/cash/backfila/dashboard/SeedDbAction.kt

```diff
@@ -0,0 +1,77 @@
+package app.cash.backfila.dashboard
```

Comment:
I'm fine with this in the test or development app. Not so much with the general service though. Maybe just break DevelopmentApp out into another module?

Reply:
I wonder if we just want to start some other test only development services. And those could have these kinds of apis.